### ISSUE TITLE
List of all kind of success and error result JSON objects

### ISSUE THREAD

**Issue by gdeverlant** [perkeep/perkeep#1042]
Hi again!

I would like to know where can I find all list of all the different kind of JSON objects returned by the HTTP API.

This includes all success and error JSON objects.

**Comment by mpl** [comment]
If you're only talking about the blobserver, we should have pretty much all of it in the docs on the website. If you're talking about Perkeep as a whole, with all its handlers, I don't think we have anything like that.

**Comment by gdeverlant** [comment]
I don't think that it is complete. The descriptions are brief there are some return value but you can't find all of it and the errors associated with them.
